content/beginner/085_scaling_karpenter/setup_the_environment.md

```diff
@@ -9,7 +9,7 @@ Before we install Karpenter, there are a few things that we will need to prepare
 ## Pre-requisites
 
 ```bash
-export CLUSTER_NAME=$(eksctl get clusters -o json | jq -r '.[0].metadata.name')
+export CLUSTER_NAME=$(eksctl get clusters -o json | jq -r '.[0].Name')
```

Comment:
I just do this module today and it was working fine. Did you had issue with this syntax ?